spec/rubocop/cop/style/multiline_assignment_layout_spec.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::MultilineAssignmentLayout, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) do
+    {
+      'SupportedTypes' => ['if', 'case']
+    }
+  end
+
+  it 'registers an offense when the rhs is on the same line' do
+    inspect_source(cop, "blarg = if true\nend")
+
+    expect(cop.offenses.length).to eq(1)
+    expect(cop.offenses.first.line).to eq(1)
+    expect(cop.highlights).to eq(["if true\nend"])
+    expect(cop.messages).to eq([described_class::MSG])
+  end
+
+  it 'auto-corrects offenses' do
+    new_source = autocorrect_source(cop, "blarg = if true\nend")
```

Comment:
I'd prefer if you used an array of lines instead of a single string with newlines, as we do in most specs.